src/ui/src/pages/app/monitors/Index/components/MonitorForm/index.tsx

```diff
@@ -50,13 +50,22 @@ const MonitorForm: React.FC = () => {
 
     setSelectedDatasource(selectedOptions);
 
-    console.log(selectedOptions);
-
     const datasource = selectedOptions[0].label;
     setDatasourceName(datasource);
   };
 
+  const validateInput = () => {
+    if(name && parseInt(intervalAmount) >= 1) {
+      // TODO: add more validators based on monitor type
+      return true;
+    }
+    return false;
+  }
```

Comment:
Don't think this is required at this stage - should instead delegate validation to server and show error if occurs on FE - can probably delete

Reply:
I do think both a FE and a server check doesn't hurt, especially at this moment where there is no verification at all...we can remove once we add server validation.